torch/linalg/__init__.py

```diff
@@ -184,3 +184,94 @@
     >>> torch.allclose(torch.tensordot(a, x, dims=x.ndim), b, atol=1e-6)
     True
 """)
+
+
+qr = _add_docstr(_linalg.linalg_qr, r"""
+qr(input, mode='reduced', *, out=None) -> (Tensor, Tensor)
+
+Computes the QR decomposition of a matrix or a batch of matrices :attr:`input`,
+and returns a namedtuple (Q, R) of tensors such that :math:`\text{input} = Q R`
```

Comment:
I think something is missing between the Q and R in the mathematical expression?

Reply:
[wikipedia](https://en.wikipedia.org/wiki/QR_decomposition) uses the very same notation (albeit without the space):
![image](https://user-images.githubusercontent.com/344507/100471331-5b6ff600-30da-11eb-8e12-606baf89fbe0.png)

Reply:
OK, sounds good.